code/modules/modular_computers/file_system/programs/antagonist/revelation.dm

```diff
@@ -15,24 +15,34 @@
 		activate()
 
 /datum/computer_file/program/revelation/proc/activate()
+	var/explosivepower = 1
 	if(computer)
+		if(computer.surgeprotected == 1)
```

Comment:
No. Maybe make this causes the computer not to explode, but keep it damaging the computer.